pytimings/timer.py

```diff
@@ -241,14 +241,32 @@ def add_extra_data(self, data: [Dict]):
 
 @contextmanager
 def scoped_timing(section_name, log_function=None, timings=None, format=''):
+    """The printout will return the added walltime for the section_name in this function call"""
```

Comment:
```suggestion
    """Start timer on entering block scope, stop it (and optionally output) on exiting. 
    
    The printout will only show walltime for the current scope. See :py:func:`~pytimings.timer.cummulative_scoped_timing` for a version with cummulative output. 
    """
```

Makes it a little clearer what this thing is for ITFP, I hope. Please check if I got the rst reference syntax right.